tests/supervisor/test_jwk.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from aiohttp import web
+
+# Avoid failures linked to settings when initializing the global VmPool object
+os.environ["ALEPH_VM_ALLOW_VM_NETWORKING"] = "False"
+
+from typing import Any, Dict
+
+import pytest
+
+from vm_supervisor.views.operator import authenticate_jwk
+
+
+@pytest.fixture
+def valid_jwk_headers(mocker):
+    mocker.patch(
+        "vm_supervisor.views.operator.is_token_still_valid", lambda timestamp: True
+    )
+    return {
+        "X-SignedPubKey": '{"payload":"7b227075626b6579223a7b22616c67223a224553323536222c22637276223a22502d323536222c22657874223a747275652c226b65795f6f7073223a5b22766572696679225d2c226b7479223a224543222c2278223a224b65763844614d7356454673365a6b4679525a4272796344564138566a334f656e49756f34743561374634222c2279223a2279597343556d715978654767673643743736794f47525873545867446444795234644f5639514c6f6b6477227d2c22616c67223a224543445341222c22646f6d61696e223a226c6f63616c686f7374222c2261646472657373223a22307833343932346566393435623933316431653932393337353535366636396365326537666535646363222c2265787069726573223a313638393337353132342e3532317d","signature":"0x58e1498a6c4f88ac1982e7147ff49405ffe1b9633e048bb74cf741abb05ce0b63bb406f3079f641ae89f597654ecd2a704d37ffbf86a28e462140033cc0eedcb1c"}',
```

Comment:
There might be an issue with the token expiration while using an hardcoded value.

Reply:
That's why I mock the expiration function in the tests.